subsys/net/lib/coap/coap.c

```diff
@@ -1228,13 +1228,14 @@ struct net_buf *coap_packet_get_payload(const struct coap_packet *cpkt,
 	u16_t coap_pkt_len;
 
 	frag = NULL;
```

Comment:
nitpick: move frag = NULL also, just for clean looking code.

Reply:
done